src/Rotations.jl

```diff
@@ -59,11 +84,29 @@ point at `y`.
 function log!(S::Rotations, v::TV, x, y) where TV
     U = transpose(x) * y
     # MMatrix doesn't have `log` defined
-    U1 = TV(real(log(Array(U))))
+    U1 = real(log(Array(U)))
```

Comment:
`log` is pretty expensive to calculate anyway, so removing a couple of allocations won't make any noticeable difference. I think switching to `_log` can be a part of #25 since it's already there.